### PR TITLE
Issue #57 fix: User cannot able to see his/her own item submissions

### PR DESCRIPTION
**Pull request by vaibhavsTekdi**
**Steps to reproduce :**

Steps to reproduce :

- Go to the admin panel. 
- Set 'View Items' ACL to 'Not Allowed' / 'Denied' of any UCM type.
- Go to the user panel & submit the item against that particular UCM type.
- Go to the item list page of that UCM type.

Here the user cannot see the list of items submitted by him/her. It shows the error/warning message like 'You are not authorized to view this resource'.

### CODE REVIEW

File: src/components/com_tjucm/site/models/items.php [hunk 1]
```diff
@@ -123,6 +124,13 @@ protected function populateState($ordering = null, $direction = null)
 		$this->setState("ucmType.id", $type_id);
 
 		$createdBy = $app->input->get('created_by', "", "INT");
+		$canView = $user->authorise('core.type.viewitem', 'com_tjucm.type.' . $type_id);
+
+		if (!$canView)
```
Comment (manojLondhe) [inline src/components/com_tjucm/site/models/items.php]: This  seems wrong, ideally, if user do not not have view access you should set createdBy as logged in user id, so he only see his own items

File: src/components/com_tjucm/site/views/items/view.html.php [hunk 2]
```diff
@@ -149,15 +148,6 @@ public function display($tpl = null)
 			$this->canView = true;
 		}
 
-		// Check the view access to the article (the model has already computed the values).
-		if (!$this->canView)
```
Comment (manojLondhe) [inline src/components/com_tjucm/site/views/items/view.html.php]: is similar check there on item view? it needs to be there

Comment (vaibhavsTekdi) [reply src/components/com_tjucm/site/views/items/view.html.php]: Yes, it's there.

File: src/components/com_tjucm/site/models/items.php (hunk 1 above)
Comment (vaibhavsTekdi) [reply src/components/com_tjucm/site/models/items.php]: Yes, I have fixed it.

**Pull request merged**